lib/diagnostic.rb

```diff
@@ -28,7 +27,9 @@ def example
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
```

Comment:
Looking for a request spec here rather than routing. Would look something like:
```
  it 'lists all examples' do
    get '/examples'

    expect(response).to be_success

    examples_response = JSON.parse(response.body)
    expect(examples_response.length).to eq(examples.count)
    expect(examples_response['examples'].first['title']).to eq(example.title)
  end
```